server/migrations/20240224200703-create-course-client.js

```diff
@@ -27,15 +27,16 @@ module.exports = {
       },
       createdAt: {
         allowNull: false,
-        type: Sequelize.DATE
+        type: Sequelize.DATE,
       },
       updatedAt: {
         allowNull: false,
-        type: Sequelize.DATE
-      }
+        type: Sequelize.DATE,
+      },
     });
+    await queryInterface.createIndex('CourseClients', ['CourseId', 'ClientId'], { unique: true });
```

Comment:
This "unique index" will prevent rows with the same CourseId, ClientId combination

Reply:
For other people who use this as reference, the command is queryInterface.addIndex and not queryInterface.createIndex

Reply:
Oops! Sorry about that...